napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -106,7 +106,9 @@ def __init__(self, layer: Layer) -> None:
             self.MODE.PAN_ZOOM,
             False,
             self.PAN_ZOOM_ACTION_NAME,
-            extra_tooltip_text=trans._('(or hold Space)'),
+            extra_tooltip_text=trans._(
+                '\n(or hold Space\n[+Shift to pan in 3D])'
+            ),
```

Comment:
```suggestion
            extra_tooltip_text=trans._(
                '\n(or hold Space)\n[In 3D hold Shift to pan]'
            ),
```
I think this is clearer? Shift works in 3D no matter how you activate the mode -- e.g. Shift-Z will work the same as Spacebar, complete with the hold-down to temporarily toggle behavior.

Reply:
This was my intent to convey! Thank you for the suggestion.